Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -859,6 +859,17 @@ def line_problems(line, _lineno)
         problem "Use `depends_on :java` to set JAVA_HOME"
       end
 
+      if line =~ /uses_from_macos (["'](.+)["'])/
+        begin
+          macos_f = Formulary.factory(Regexp.last_match(2))
+          unless macos_f.keg_only_reason&.reason == :provided_by_macos
```

Comment:
I'm not sure this is quite true for now; you there's definitely things provided by macOS which aren't keg-only.

Reply:
True. We could maintain a white list of these formulae, if we wanted to go this way.

Reply:
Yes.